akka-actor/src/main/resources/reference.conf

```diff
@@ -567,7 +567,28 @@ akka {
       # enable WARN logging of misconfigured routers
       router-misconfiguration = off
     }
-
+    
+    # SECURITY BEST-PRACTICE is to disable java serialization for its multiple 
+    # known attack surfaces. 
+    # 
+    # This setting is a short-cut to
+    # - using DisabledJavaSerializer instead of JavaSerializer
+    # - enable-additional-serialization-bindings = on
+    #
+    # Completely disable the use of `akka.serialization.JavaSerialization` by the 
+    # Akka Serialization extension, instead DisabledJavaSerializer will 
+    # be inserted which will fail explicitly if attempts to use java serialization are made.
+    # 
+    # The log messages emitted by such serializer SHOULD be be treated as potential 
+    # attacks which the serializer prevented, as they MAY indicate an external operator 
+    # attempting to send malicious messages intending to use java serialization as attack vector.
+    # The attempts are logged with the SECURITY marker.
+    # 
+    # Please note that this option does not stop you from manually invoking java serialization
+    # 
+    # The default value for this might be changed to off in future versions of Akka.
```

Comment:
If we do I think we must present a good alternative. The only rather seamless alternative I know of is Kryo and then I think we must take responsibility of that serializer.